From 88c2e7dcf8adff93a335418390fb5f23f83e4f3c Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Mon, 23 Jul 2012 17:36:03 -0400
Subject: [PATCH] systemd: more updates to start service

The dashes next to the semi-colons seem to confuse systemd so drop them.
---
 systemd-units/plymouth-start.service.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: plymouth-0.8.6.1/systemd-units/plymouth-start.service.in
===================================================================
--- plymouth-0.8.6.1.orig/systemd-units/plymouth-start.service.in
+++ plymouth-0.8.6.1/systemd-units/plymouth-start.service.in
@@ -9,5 +9,5 @@ ConditionPathExists=!@plymouthruntimedir
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=boot --pid-file=@plymouthruntimedir@/pid --attach-to-session
-ExecStartPost=-/bin/udevadm settle --timeout=30 --exit-if-exists=/sys/class/drm/card0/dev;-/bin/udevadm settle --timeout=30 --exit-if-exists=/sys/class/graphics/fb0/dev; -@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
+ExecStartPost=-/bin/udevadm settle --timeout=30 --exit-if-exists=/sys/class/drm/card0/dev ; /bin/udevadm settle --timeout=30 --exit-if-exists=/sys/class/graphics/fb0/dev ; @PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
